api/base/views.py

```diff
@@ -6,57 +6,214 @@
+    failed without the attribute update, the API will still report a failure.
 
+    Typoed or non-existant attributes will behave the same as non-updatable attributes and be silently
+    ignored. If a request is not working the way you expect, make sure to double check your spelling.
 
-        The OSF stores, documents, and archives study designs, materials, data, manuscripts, or anything else associated
-        with your research during the research process. Every project and file on the OSF has a permanent unique
-        identifier, and every registration (a permanent, time-stamped version of your projects and files) can be assigned
-         a DOI/ARK. You can use the OSF to measure your impact by monitoring the traffic to projects and files you make
-         public. With the OSF you have full control of what parts of your research are public and what remains private.
+    ###Errors
 
-        Beta notice: This API is currently a beta service.  You are encouraged to use the API and will receive support
-        when doing so, however, while the API remains in beta status, it may change without notice as a result of
-        product updates. The temporary beta status of the API will remain in place while it matures. In a future
-        release, the beta status will be removed, at which point we will provide details on how long we will support
-        the API V2 and under what circumstances it might change.
+    When a request fails for whatever reason, the OSF API will return an appropriate HTTP error code and include a
+    descriptive error in the body of the response.  The response body will be an object with a key, `errors`, pointing
+    to an array of error objects.  Generally, these error object will consist of a `detail` key with a detailed error
+    message, but may include additional information in accordance with the [JSON-API error
+    spec](http://jsonapi.org/format/1.0/#error-objects).
 
-        ##General API Usage
-        Each endpoint will have its own documentation, but there are some general principles.
+    ##OSF Enum Fields
 
-        ###Filtering
-        Collections can be filtered by adding a query parameter in the form:
+    Some entities in the OSF API have fields that only take a restricted set of values.  Those fields are listed here
+    for reference.  Fuller descriptions are available on the relevent entity pages.
 
-            filter[<fieldname>]=<matching information>
-        For example, if you were trying to find [Lise Meitner](http://en.wikipedia.org/wiki/Lise_Meitner):
+    ###OSF Node Categories
 
-            /users?filter[fullname]=meitn
-        You can filter on multiple fields, or the same field in different ways, by &-ing the query parameters together.
+        value                 description
+        ------------------------------------------
+        project               Project
```

Comment:
Only in that it's an option that is listed in the enum but not here. And it's the default value. But I don't think it hurts anything leaving it out.